crates/bytesbuf/src/span_builder.rs

```diff
@@ -185,6 +180,37 @@ impl SpanBuilder {
         Some(span)
     }
 
+    /// Creates a span over the filled data without consuming it from the builder.
+    /// The span shares the BlockRef with the span builder, keeping the memory alive.
```

Comment:
This line about keeping memory alive seems confusing and unnecessary. Obviously, if we reference some data in some memory, that memory will be "alive". This says nothing useful.